thirdparty/build-thirdparty.sh

```diff
@@ -566,6 +566,19 @@ build_flatbuffers() {
   cp libflatbuffers.a $TP_LIB_DIR/libflatbuffers.a
 }
 
+build_brotli() {
+    check_if_source_exist $BROTLI_SOURCE
+    cd $TP_SOURCE_DIR/$BROTLI_SOURCE
+    mkdir -p $BUILD_DIR
+    cd $BUILD_DIR
+    ${CMAKE_CMD} .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR
+    ${CMAKE_CMD} --build . --config Release --target install
+
+    cp -rf $TP_INSTALL_DIR/lib/libbrotlienc-static.a $TP_INSTALL_DIR/lib64/libbrotlienc.a
+    cp -rf $TP_INSTALL_DIR/lib/libbrotlidec-static.a $TP_INSTALL_DIR/lib64/libbrotlidec.a
+    cp -rf $TP_INSTALL_DIR/lib/libbrotlicommon-static.a $TP_INSTALL_DIR/lib64/libbrotlicommon.a
```

Comment:
I would rather use `mv` than `cp`, don't want multiple copies in the target install location.

Reply:
also need to remove line 629~631, since we do it separately, so no need to ask arrow to do it again.